pkg/gofr/datasource/cassandra/cassandra.go

```diff
@@ -3,9 +3,12 @@ package cassandra
 import (
 	"context"
 	"errors"
+	"fmt"
 	"reflect"
 	"time"
 
+	"go.opentelemetry.io/otel/attribute"
+
 	"go.opentelemetry.io/otel/trace"
```

Comment:
similar type of imports.. can be clubbed together

Reply:
sure done